<?xml version="1.0"?>
<launch>
  <arg name="num_waypts" default="10" />
  <arg name="chain_start" default="link0" />
  <arg name="chain_end" default="link7" />
  <arg name="timeout" default="0.005" />
   <!-- be sure to launch the torobo launch file first -->
  <node name="trac_ik_torobo" pkg="trac_ik_torobo" type="ik_torobo" output="screen">
    <param name="num_waypts" value="$(arg num_waypts)"/>
    <param name="chain_start" value="$(arg chain_start)"/>
    <param name="chain_end" value="$(arg chain_end)"/>
    <param name="timeout" value="$(arg timeout)"/>
    <param name="urdf_param" value="/robot_description"/>
  </node>


</launch>
